README.md

```diff
@@ -0,0 +1,4 @@
+to build the todo-app containers run
+```bash
+./build-todo-app.sh <version> <repo>
```

Comment:
nit: Iirc, our other scripts to build stuff tend to either use environment variables or script options. Could we do those here (if not a big hassle to change everything). I _personally_ think it would be a bit more user-friendly, instead of just passing two values where order matters like here.

ex.
`VERSION="1.0" IMAGE_REPO="repo" ./build-todo-app.sh`
or
`./build-todo-app.sh -v 1.0 --repo jackstine`